### ISSUE TITLE
Hovering over Thaumcraft aspects in NEI bookmark list freezes the game

### ISSUE THREAD

**Issue by AbdielKavash** [GTNewHorizons/GT-New-Horizons-Modpack#14292]
### Your GTNH Discord Username

abdiel_kavash

### Your Pack Version

2.3.6

### Your Server

SP

### Java Version

Java 19

### Type of Server

Single Player

### Your Expectation

I am using the NEI shortcut ctrl+shift+A to bookmark a recipe using the crucible or arcane infusion. This in addition to the ingredient items adds the aspects required for the craft to the bookmark.

### The Reality

When I move my mouse over the aspect icons in the bookmark list, the game freezes for several seconds. I suspect this is because the game is trying to load this view:

![](https://i.imgur.com/ZLyk04V.png)

and since I had scanned many items, it is taking a while to populate. Indeed, trying to look up the sources of an aspect in NEI itself also freezes the game like this (and in my previous world where I had scanned even more items, it could freeze for a minute or more).

### Your Proposal

While the freeze itself when looking up the aspect sources NEI page might be unavoidable (I do not know what is causing it), it would be nice if accidentally hovering over the aspect icon in the bookmark panel would not make NEI try to load that page. If I specifically need to look up the aspects I can tolerate the freeze, but it is very annoying when I am doing something else and it is triggered by simply wanton mouse movement.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by desagas** [comment]
It momentarily slows it down on my computer, but as soon as it loads the items that can create it, it resumes fine, it does not freeze it. I assume this may be quite slow on slower computers.

I wish this view pane that moves everywhere your mouse moves was not default, but a button or a shift or something else could be used to toggle it on.

**Comment by AbdielKavash** [comment]
Yes, it does not freeze forever, only until the NEI page loads. Still, if you chronically scan everything you see, it can take far longer than comfortable even on a decent PC.

**Comment by desagas** [comment]
It will only freeze on each aspect once, I believe, as it loads it into memory. From there, I have never myself experienced it lagging every single time you hover over the same aspect? Does this happen to you everytime? I can even exit the crafting or inventory screen, and do stuff and come back, and still no additional lag.
